iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/HelmProcessor.java

```diff
@@ -81,6 +86,11 @@ String processHelmTemplate(String path, String source, InputFileContext inputFil
     return evaluateHelmTemplate(path, inputFileContext.inputFile, sourceWithComments, fileContents);
   }
 
+  @CheckForNull
+  public Tree getGoAstForInputFile(InputPath inputPath) {
+    return inputFileToGoAst.remove(inputPath.uri().toString());
```

Comment:
Why here is `remove` instead of `get`?

Reply:
As we now don't have any caching, I wanted to avoid these trees staying in memory after we are done processing the current file. I'll add a comment in the code